app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -159,4 +165,55 @@ public function getStoreIds()
         return $storeIds;
     }
 
+    private function getOrderProducts(Mage_Sales_Model_Order $order): array {
+        $products = [];
+        foreach ($order->getAllItems() as $item) {
+            $product = $item->getProduct();
+            if ($product->isConfigurable()) {
+                continue;
+            }
+            $products[] = [
+                'name' => $product->getName(),
+                'url' => $this->getProductUrl($product),
+                'id' => $product->getId(),
+                'sku' => $product->getSku(),
+                'image_url' => $this->getProductImageUrl($product),
+                'brand' => $product->getAttributeText('manufacturer'),
+            ];
+        }
+
+        return $products;
+    }
+
+    private function getProductImageUrl(Mage_Catalog_Model_Product $product): string {
+        if ($product->getMediaGalleryImages()->getSize() < 1) {
```

Comment:
You can reduce nesting in this function by swapping this comparison around:

```suggestion
        if ($product->getMediaGalleryImages()->getSize()) {
          return $product->getImageUrl();
        }
```